docs/prow/k3d-no-ssh.md

```diff
@@ -0,0 +1,119 @@
+# Run K3d cluster inside ProwJobs
+
+This document provides simple instruction with examples on how to prepare ProwJob to use K3d cluster and Docker.
+
+> This guide also applies for [kind](https://kind.sigs.k8s.io) and other workloads that require Docker.
+
+ProwJob workload operates on a Pod, Kubernetes' primitive resource that orchestrates OCI or Docker images. That means
+every ProwJob can be configured the same way as any Pod. K3d is a CLI tool that runs K3s cluster inside Docker daemon.
+Since it's possible to run Docker daemon inside Kubernetes' Pod (Docker in Docker) this can also be used to run K3s cluster in it.
+
+Fortunately, in Prow configuration there are 2 useful presets that configure ProwJob to use Docker inside. Those 2 presets can be found in [prow/config.yaml](../../prow/config.yaml).
+
+```yaml
+labels:
+  preset-kind-volume-mounts: "true"
+  preset-dind-enabled: "true"
+```
+
+Additionally, ProwJob's container needs to be run in `privileged` mode, since it requires to mount host's system paths `/sys/fs/cgroup` and `/lib/modules` to be able to spawn Docker daemon.
```

Comment:
```suggestion
Additionally, the ProwJob's container needs to be run in the `privileged` mode since it requires mounting host system paths `/sys/fs/cgroup` and `/lib/modules` to be able to spawn the Docker daemon.
```